content/en/monitors/incident_management/_index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Incident Management
+kind: documentation
+disable_toc: true
+description: Create and manage incidents
+---
+
+{{< img src="monitors/incidents/incidents-top-1.png" alt="Incident Management"  style="width:80%;">}}
+
+Any event that may lead to a disruption in your organization’s services can be described as an incident, and it is often necessary to have a set framework for handling these events. Datadog’s Incident Management feature provides a system through which your organization can effectively identify and mitigate incidents. 
+
+Incidents live in Datadog alongside the metrics, traces, and logs you are collecting. You can view and filter incidents that are relevant to you.
+
+In the Datadog paradigm, any of the following are appropriate situations for declaring an incident:
+
+* An issue is or may be impacting customers or services.
+* You do not know whether you should call an incident. Notify other people and increase severity appropriately.
+
+## Usage
+
+Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing incidents.  You can configure additional fields that appear for all incidents in [Incident Settings][2].
+
+### Creating an incident
+
+#### From a graph
+
+You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph" style="width:80%;">}}
+
+#### From the clipboard
+
+Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the clipboard and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.
+
+{{< img src="monitors/incidents/from-clipboard.png" alt="Add a dashboard to the clipboard" style="width:80%;">}}
+
+{{< img src="monitors/incidents/clipboard.png" alt="Create in incident from the clipboard" style="width:80%;">}}
+
+**Note**: In addition to exporting from an incident, data on the clipboard can be exported to a new dashboard or a notebook.
+
+#### From a monitor
+
+You can declare an incident directly from a monitor by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the monitor is added into the incident as a signal.
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a monitor" style="width:80%;">}}
+
+#### From the incidents page
+
+In the [incidents UI][1], click the **New Incident** button to create an incident.
+
+#### From Slack
+
+Once you have the [Datadog integration enabled on Slack][3], you can use `/datadog` to declare a new incident from any channel.
```

Comment:
Once you have the [Datadog integration enabled on Slack][3], from any Slack channel you can use the slash command `/datadog incident` to declare a new incident. 

In the creation modal, you add a descriptive title, select whether customers were impacted (yes, no, or unknown) and select a severity level (1-5, unknown). 

If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user will become the Incident Commander. The Incident Commander (IC) can be changed later in-app if necessary. If the person declaring an incident is not a member of a Datadog account, then the IC is assigned to a generic `Slack app user` and can be assigned to another IC in-app.

Read more about using the Datadog Slack App here.

Reply:
Updated with Natalie's comments in the draft doc

Reply:
completely missed these, thanks!